### PR TITLE
Add 'archive' icon

### PR DESCRIPTION
**Pull request by AdrianoFerrari**

### CODE REVIEW

File: src/Octicons.elm [hunk 1]
```diff
@@ -41,7 +41,7 @@ module Octicons exposing (..)
 @docs pathIconWithOptions, polygonIconWithOptions, Options, defaultOptions
 
 # Svg Data
-@docs alertPath, arrowDownPolygon, arrowLeftPolygon, arrowRightPolygon, arrowSmallDownPolygon
+@docs alertPath, archiveData arrowDownPolygon, arrowLeftPolygon, arrowRightPolygon, arrowSmallDownPolygon
```
Comment (YetAnotherMinion) [inline src/Octicons.elm]: should be `archivePath` and add comma.

Test that your changes are valid by building the library and making sure there are no errors.
```bash
elm-make src/Octicons.elm
elm-make --docs foo.json
```

File: src/Octicons.elm [hunk 2]
```diff
@@ -187,6 +187,20 @@ alertPath =
     "M8.865,1.51999998 C8.685,1.20999998 8.355,1.01999998 7.995,1.01999998 C7.635,1.01999998 7.305,1.20999998 7.125,1.51999998 L0.275000001,13.5 C0.0950000006,13.81 0.0950000006,14.19 0.275000001,14.5 C0.465000001,14.81 0.795000001,15 1.145,15 L14.845,15 C15.205,15 15.535,14.81 15.705,14.5 C15.875,14.19 15.885,13.81 15.715,13.5 L8.865,1.51999998 Z M8.995,13 L6.995,13 L6.995,11 L8.995,11 L8.995,13 L8.995,13 Z M8.995,9.99999998 L6.995,9.99999998 L6.995,5.99999998 L8.995,5.99999998 L8.995,9.99999998 L8.995,9.99999998 Z"
 
 
+{-|
+-}
+archive : Options -> Html msg
+archive =
+    pathIconWithOptions archivePath "0 0 16 16" "archive"
```
Comment (YetAnotherMinion) [inline src/Octicons.elm]: Viewbox should be "0 0 14 16", See official Octicons definition https://octicons.github.com/icon/archive/

**Comment by YetAnotherMinion** [comment]
Thank you for your contribution, if I could figure out how to edit your PR from the web UI I would make changes myself. Maybe there is a checkbox you can hit to allow maintainers to edit your PR?

**Comment by AdrianoFerrari** [comment]
Thanks for the detailed feedback. Making those changes now and will push soon.

**Comment by AdrianoFerrari** [comment]
Ok, @YetAnotherMinion 
Made the changes, and checked that `--docs` and `tests/Main.elm` both work.
Let me know if there's anything else.

PS: `elm-make src/Octicons.elm` yields this error, but does so even on master branch:
```
Uncaught TypeError: Elm.Octicons.fullscreen is not a function
```

**Comment by YetAnotherMinion** [comment]
Package version 2.2.0 published. Thanks again for the fast turnaround.

**Comment by AdrianoFerrari** [comment]
Thank **you** for the fast turnaround as well :+1:

**Pull request merged**